src/i18n/en.json

```diff
@@ -293,6 +293,9 @@
         "data-portability": "Get data in a machine-readable format",
         "erase-all": "Erase all data",
         "erasure-data": "Data to erase",
+        "include-objection": "Additionally include an objection",
```

Comment:
I think we need to explain here against what the objection is. Like "Additionally object against interests of the controller"

Reply:
```suggestion
        "include-objection": "Additionally include an objection based on your particular situation",
```